mentoring/mentees/ResourcesForMentees.md

```diff
@@ -0,0 +1,139 @@
+
+- **LFX Mentorship**: Learn about opportunities to engage with mentorship programs supported by the Linux Foundation, connecting mentees with experienced mentors across various projects.
+- **Link**: [LFX Mentorship](https://lfx.linuxfoundation.org/tools/mentorship/)
+
+- **Linux Foundation Training & Certification**: Explore a variety of courses and certifications that cater to different skill levels and career paths in open source.
+  - [Linux Foundation Training](https://training.linuxfoundation.org/). Consider their FREE [Cybersecurity Essentials Certification](https://training.linuxfoundation.org/training/cybersecurity-essentials-lfc108/). 
+
+# Understanding Linux Foundation Special Interest Groups (SIGs)
+
+Special Interest Groups (SIGs) within the Linux Foundation are collaborative groups that focus on specific areas of interest in the broader landscape of open-source projects and technologies. SIGs play a crucial role in fostering innovation, sharing knowledge, and working on common goals within their respective domains. Members of SIGs include industry professionals, developers, researchers, and anyone passionate about contributing to the advancement of open-source technologies.
+
+## SIGs in the Confidential Computing Consortium (CCC)
+
+The Confidential Computing Consortium (CCC) supports several SIGs focused on different aspects of confidential computing. These groups work on initiatives such as developing open standards, creating reference architectures, and enhancing the security and usability of confidential computing technologies. Here are the SIGs associated with the CCC and how you can get involved:
+
+### CCC Special Interest Groups
+
+## Outreach Committee
+
+**Focus**: The Outreach Committee is responsible for designing and executing community outreach efforts to maximize the outreach and visibility of the CCC’s efforts in confidential computing.
+
+**Meeting Schedule**: Bi-weekly on Wednesdays at 8:00 AM Pacific Time.
+
+## Technical Advisory Council (TAC)
+
+**Focus**: The TAC acts as the technical resource arm of the CCC, facilitating communication among community projects and nurturing the technical vision for the community.
+
+**Meeting Schedule**: Bi-weekly on Thursdays at 7:00 AM Pacific Time.
+
+### TAC Subcommittee: Attestation Special Interest Group (Attestation-SIG)
+
+**Focus**: This SIG is dedicated to enhancing interoperability and authentication through attestation in confidential computing environments.
+
+**Meeting Schedule**: Bi-weekly on Tuesdays at 9:00 AM Pacific Time.
+
+### TAC Subcommittee: Governance Risk & Compliance Special Interest Group (GRC-SIG)
+
+**Focus**: The GRC SIG works to connect the confidential computing development community with regulatory framework creators, aiming to craft secure and compliant regulations and governance patterns.
+
+**Meeting Schedule**: Weekly on Wednesdays at 7:00 AM Pacific Time.
+
```

Comment:
Waiting on Dan, Riaan to confirm meeting details for the SIG, which can be updated later. 
Riaan will update CCC website with the same.

Reply:
Maybe better we just link to the website for each committee and SIG... following principle of maintaining info in one place.

Reply:
@dcmiddle need more info from you about how this section should look like. So I have left it as is for now.

Reply:
Sorry for the ambiguity. I mean that instead of writing a blurb for each committee/sig we just refer to the website. That way when we change the times for those meetings or add new SIGs we only update the website. We don't need to remember to update info in lots of other places like this. Something like ...
"... and enhancing the security and usability of confidential computing technologies.
You can read about each SIG including how to learn and contribute on our website https://confidentialcomputing.io/about/committees/"

Reply:
Thanks @dcmiddle .
Updated in latest commit.